app/src/main/java/org/orpheus/App.java

```diff
@@ -11,5 +11,6 @@ public String getGreeting() {
 
   public static void main(String[] args) {
     System.out.println(new App().getGreeting());
+    new RecordAudio().hello("This class will handle recording desktop audio.");
```

Comment:
This commit is about fixing the file structure. Why are you changing this line here?